### DISCUSSION TITLE
Can not switch model from the right side bar in mobile version

### DISCUSSION THREAD

**Discussion by Th3X-Zohir** [danny-avila/LibreChat#3032] [Troubleshooting]
When you click to the right side bar menu and try to select a model it does not let you to select it there is some ui issues here

https://github.com/danny-avila/LibreChat/assets/60055496/5d97966b-7a14-4c28-b506-eb53d99e6f74

**Comment by danny-avila** [comment]
Must be specific to your phone's OS/browser as I can't reproduce it below. Maybe @berry-13 can look into it who has an android device.

### desktop (chrome)
https://github.com/danny-avila/LibreChat/assets/110412045/7e74edaf-12b7-46bd-91ca-4e982784f237


### iPhone (PWA)
https://github.com/danny-avila/LibreChat/assets/110412045/137083e8-8934-4ea8-90fa-3960fb422758

### iPhone (Chrome)
https://github.com/danny-avila/LibreChat/assets/110412045/2719d4cd-3f63-4c96-9fae-33883d91fb3d

**Comment by berry-13** [comment]
@danny-avila yes, I'm also having this issue. Same issue on Samsung Browser, Chrome and Brave (mobile)

**Reply by danny-avila** [reply]
Alright thanks for confirming. Please look into it when you have the time as I can’t reproduce it on any of the devices I have.
